107/src/main/java/org/ehcache/jsr107/ConfigurationMerger.java

```diff
@@ -162,6 +164,43 @@
     return builder;
   }
 
+  private static <K, V> CacheConfigurationBuilder<K, V> addDefaultCopiers(CacheConfigurationBuilder<K, V> builder, Class keyType, Class valueType ) {
```

Comment:
Yeah, you're damn right. Ignore what I said, I just remembered that we changed the way to configure store-by-value in ehcache to configure copiers.